api/src/main/java/org/datacleaner/api/OutputColumns.java

```diff
@@ -30,10 +30,24 @@
 
     private static final long serialVersionUID = 1L;
 
+    /**
+     * Constant {@link OutputColumns} values for components with no/zero output
+     * columns.
+     */
+    public static final OutputColumns NO_OUTPUT_COLUMNS = new OutputColumns();
+
     private final String[] columnNames;
     private final Class<?>[] columnTypes;
 
     /**
+     * Private constructor used to create an empty output columns objects.
+     */
+    private OutputColumns() {
+        columnNames = new String[0];
+        columnTypes = new Class[0];
+    }
```

Comment:
This is for the constant above (NO_OUTPUT_COLUMNS). It is used by MultiStreamComponent which is technically a transformer, but without output columns.